### ISSUE TITLE
Did it work?

### ISSUE THREAD

**Issue by cortezcristian** [notch8-old/mobiscroll_and_ionic#1]
After looking at your video it looks like you wasn't able to make this work.

**Comment by winescout** [comment]
no, we ended up breaking it up to use a date, and separate time selector, and then using the native stuff.  That was a year ago or so, things likely are different now.
